data/content_scripts/posting_process/iframe.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileOverview Script used to post the privly URL into the Host page target node.
+ */
+function dispatchClickEvent(target) {
+  var evt = new MouseEvent("click", {
+    bubbles: true,
+    cancelable: true,
+    view: document.defaultView,
+  });
+  target.dispatchEvent(evt);
+}
+
+self.port.on("postURL", function(message) {
```

Comment:
Separating `self.port.on` from the function will be a consistent need across the code base.